vms/platformvm/txs/builder/camino_builder.go

```diff
@@ -683,6 +690,58 @@ func (b *caminoBuilder) NewSystemUnlockDepositTx(
 	return tx, tx.SyntacticVerify(b.ctx)
 }
 
+func (b *caminoBuilder) NewMultisigAliasTx(
+	multisigAlias multisig.Alias,
+	keys []*crypto.PrivateKeySECP256K1R,
+	change *secp256k1fx.OutputOwners,
+) (*txs.Tx, error) {
+	ins, outs, signers, _, err := b.Lock(keys, 0, b.cfg.TxFee, locked.StateUnlocked, nil, change, 0)
+	if err != nil {
+		return nil, fmt.Errorf("couldn't generate tx inputs/outputs: %w", err)
+	}
+
+	// Update existing multisig alias definition
+	var sigIndices []uint32
+
+	if multisigAlias.ID != ids.ShortEmpty {
+		addrs := []ids.ShortID{multisigAlias.ID}
```

Comment:
addrs implies that there are multiple addresses, suggest to rename it, its really confusing